pkg/organizations/client.go

```diff
@@ -68,6 +68,12 @@ type Organization struct {
 
 	// The Organization's Domains.
 	Domains []OrganizationDomain `json:"domains"`
+
+	//Created At
```

Comment:
Similar to @maxdeviant's previous comment, we should update these comments (and the ones in `pkg/sso/client.go`).

After that, I think this will be good to go!

Reply:
Hopefully got it all :)

Reply:
Looks like there's just a few more for `Organization` in `pkg/organizations/client.go`.

Reply:
Brutal that I missed that but this is now updated in: 

`pkg/sso/client.go`
`pkg/organizations/client.go`
`pkg/directorysync/client.go `